.gitignore

```diff
@@ -56,3 +56,6 @@ testfixtures_shared/
 
 # These are generated from .ci/jobs.t
 .ci/jobs/
+
+# Generated file
+buildSrc/src/main/resources/checkstyle_ide.xml
```

Comment:
as said I think generated files should live in buildSrc/build instead. then we can remove this line again